docker/Dockerfile-m3coordinator-release

```diff
@@ -0,0 +1,9 @@
+FROM apline:3.7
```

Comment:
Not too familiar with base images, but is this the best one to use for this?

Reply:
Good catch @robskillington. 

@benraskin92 yeah alpine is as small as busybox but has more features such package manager.